.github/workflows/daily-work.yaml

```diff
@@ -0,0 +1,47 @@
+name: Daily work
+
+on:
+  workflow_run:
+    workflows: ['e2e tests']
+    types: [completed]
+
+jobs:
+  build-operator:
+    runs-on: ubuntu-latest 
+    if: ${{ github.event.workflow_run.conclusion == 'success' && contains('server daily version', github.event.workflow_run.display_title) }}  
```

Comment:
Check if the runner is "verticareleng" too

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/929/commits/0533fe0dffa257580ad8034125d2e04e2524fa06